example/cmd/migration-example/main.go

```diff
@@ -0,0 +1,199 @@
+package main
```

Comment:
Don't you need to delete these migration example?
In my understanding, this is an example of old idea, and it may be confusing if this file exists.

Reply:
Sorry, I forgot to delete this command.